apps/extension/src/routes/popup/index.tsx

```diff
@@ -24,26 +26,21 @@ export const popupIndexLoader = async () => {
 };
 
 export const PopupIndex = () => {
```

Comment:
Something changed, I think here, where we lost our secondary font kind. Looks like all text is now `Faktum`. We should figure out what did that. Less important text should be the secondary font.